db/query.go

```diff
@@ -76,7 +76,8 @@ var QueryChannels = SGQuery{
 			"FROM `%s` "+
 			"USE INDEX ($idx) "+
 			"UNNEST OBJECT_PAIRS($sync.channels) AS op "+
-			"WHERE [op.name, LEAST($sync.sequence, op.val.seq),IFMISSING(op.val.rev,null),IFMISSING(op.val.del,null)]  BETWEEN  [$channelName, $startSeq] AND [$channelName, $endSeq] "+
+			"WHERE $activeOnlyFilter [op.name, LEAST($sync.sequence, op.val.seq),IFMISSING(op.val.rev,null),"+
```

Comment:
The "$param" syntax is used for parameter replacement by N1QL in query predicates.  This might lead to confusion here between the usage for $activeOnlyFilter (string replacement) and the usage for $channelName, $startSeq, $endSeq (N1QL parameters).

I'd suggest switching to $$activeOnlyFilter, like we're doing for $$selectUserName.

Reply:
Yeah. make sense. Added the changes to make the placeholder as $$activeOnlyFilter.